### PR TITLE
UHM-5817: validate format of Haiti phone number

### PR DESCRIPTION
**Pull request by mogoodrich**

### CODE REVIEW

File: configuration/pih/pih-config-haiti.json [hunk 1]
```diff
@@ -70,6 +70,18 @@
           "mothersName": {
             "required": true
           }
+        },
+        "contactInfo": {
+            "phoneNumber": {
+              "regex": "(^\\d{8}$)|(^\\d{4}(?:\\)|[-|\\s])?\\s*?\\d{4}$)"
```
Comment (mogoodrich) [inline configuration/pih/pih-config-haiti.json:76]: I just combined the two regex from the zl.js.  I tested and it appears to work, but @louidorjp @rjacques89  would be good to review.

File: configuration/pih/pih-config-haiti.json [hunk 2]
```diff
@@ -70,6 +70,18 @@
...
+            }
+        },
+        "contactPerson": {
```
Comment (mogoodrich) [inline configuration/pih/pih-config-haiti.json:79]: I had to add support in PIH Core for adding regex support "contactPerson", that change will be coming in another PR.

**Review by mseaton — APPROVED** [review]
